autotest/ogr/ogr_pgeo.py

```diff
@@ -286,7 +286,101 @@ def test_ogr_pgeo_8():
     feat_count = non_spatial_layer.GetFeatureCount()
     assert feat_count == 2, 'did not get expected feature count'
 
+##################################################################################
+# Open mdb with polygon layer containing a mix of single and multi-part geometries
+
+
+def test_ogr_pgeo_9():
+    if ogrtest.pgeo_ds is None:
+        pytest.skip()
+
+    if ogrtest.active_driver.GetName() != 'PGeo':
+        # MDB driver doesn't report non-spatial tables
+        pytest.skip()
+
+    ogrtest.pgeo_ds = ogr.Open('data/pgeo/mixed_types.mdb')
+    if ogrtest.pgeo_ds is None:
+        pytest.skip('could not open DB. Driver probably misconfigured')
+
+    polygon_layer = ogrtest.pgeo_ds.GetLayerByName('polygons')
+    assert polygon_layer.GetGeomType() == ogr.wkbMultiPolygon
+
+    feat = polygon_layer.GetNextFeature()
+    if feat.GetField('pk') != 1:
```

Comment:
you can probably remove testing for the pk field here. it is not relevant for the purpose of the test. applies to similar below instances

Reply:
My intention there was to make the test easier to debug if (for whatever reason) the ordering of features changes

Reply:
ok, perhaps just a ``assert feat.GetField('pk') == 1`` would be a good compromise of expressiveness vs verbosity (I like succint tests :-))